packages/web/src/components/instagram-auth/InstagramAuth.tsx

```diff
@@ -88,10 +89,14 @@ const InstagramAuth = ({
         }, {})
 
         return onSuccess(profileRespJson.username, igUserProfile)
-      } catch (err) {
-        console.error(err)
+      } catch (err: any) {
+        reportToSentry({
+          level: ErrorLevel.Error,
```

Comment:
Is this not the default?

Reply:
It was not, but I just changed it to be so cause thats way better haha